### ISSUE TITLE
Failure to bootstrap on Windows 7 64-bit

### ISSUE THREAD

**Issue by azawawi** [tadzik/panda#182]
I encountered this error on my development windows box. The path ```C:\rakudo/share/perl6/site/bin``` is not even there after a fresh rakudo build. Even after creating it, the error stays the same. Please advise.

```
d:\tools\panda>perl6 rebootstrap.pl
No need to rebootstrap, running normal bootstrap
==> Bootstrapping Panda
==> Fetching File::Find
==> Building File::Find
Compiling lib\File\Find.pm to mbc
==> Testing File::Find
===SORRY!===
no such file or directory
==> Please make sure that C:\rakudo/share/perl6/site/bin is in your PATH
```

**Comment by ugexe** [comment]
http://testers.perl6.org/reports/54995.html

    Failed to copy 'C:\Users\Nick\AppData\Local\Temp\panda\t\fakestate' to 'REMOVEME': Failed to copy file: illegal operation on a directory
    in any  at C:\Users\Nick\Documents\GitHub\rakudobrew\moar-nom\install/share/perl6/runtime/CORE.setting.moarvm:1
    in block  at t\ecosystem.t:9

**Comment by azawawi** [comment]
Even after fixing it, panda.bat is not there in ```C:/rakudo/share/perl6/site/bin```. Is it a rakudo related bug?

**Comment by ugexe** [comment]
Considering `panda` depends on `File::Find` and `File::Find` doesn't pass on windows, i'm not sure how you are even getting that far. http://testers.perl6.org/dist/F/File::Find/%3Cunknown%3E.html

**Comment by azawawi** [comment]
@ugexe Windows 7 Ultimate 64-bit (version: ```6.1```) is passing here

The one that is failing is either Windows Server 2012 R2 or Windows 8.1. The funny thing is that ```$*DISTRO.version``` is also ```v6.3``` which is plain wrong.

Please see:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms724832%28v=vs.85%29.aspx

**Comment by ugexe** [comment]
Is it actually being precompiled and tested against that? It may be worth it to look at the actual results to make sure a segfault is not making it ignore pass a failing test (which does happen). Or is it possible you are only testing the non-precompiled source or have an old (and theoretically working) version of File::Find installed already?

**Comment by ugexe** [comment]
@azawawi it looks like (in your case anyway, not mine) it was a rakudo bug related to list flattening that was not previously needed.

https://github.com/rakudo/rakudo/pull/467

**Comment by azawawi** [comment]
This has been fixed in rakudo/rakudo#467 and #183. Closing this issue. Thanks for your time :+1:
